eox_tagging/test/test_models.py

```diff
@@ -318,18 +318,49 @@ def test_create_without_default_owner(self):
                 "tag_type": "example_tag_3",
                 "validate_tag_value": {"regex": r".*eduNEXT$"},
                 "validate_target_object": "CourseEnrollment",
-                "validate_expiration_date": {"exist": True, "in": ["Dec 04 2020", "Oct 19 2020"]},
-                "validate_activation_date": "Jun 16 2020"
+                "validate_expiration_date": {"exist": True, "between": ["2020-10-19 10:20:30", "2020-12-04 10:20:30"]},
```

Comment:
Here we have two successful tests,but  is there a test when the date is  not in the list ? and could you add one when the data is not between the values

Reply:
Done!